presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/QueryStatistics.java

```diff
@@ -73,6 +74,7 @@ public QueryStatistics(
             int completedSplits,
             boolean complete,
             List<ResourceDistribution> cpuTimeDistribution,
+            List<ResourceDistribution> peakMemoryDistribution,
```

Comment:
What about `taskPeakMemoryDistribution`? No strong opinion since `peakMemoryDistribution` is parallel to `cpuTimeDistribution`